### ISSUE TITLE
Converting gps coordinates to address

### ISSUE THREAD

**Issue by atao** [picklepete/pyicloud#168]
I suggest to add conversion of gps coordinates in adress with Geopy, good idea?

**Comment by PythonicCoder101** [comment]
It's a great idea. I did it a few weeks ago and it works perfectly!!

**Comment by coddingtonbear** [comment]
Yep!  That's a neat thing that geopy lets you do.  I'm really not sure that such a conversion should really be in this library, though -- the library's core focus is interacting with Apple's iCloud service, and adding that kind of thing is easy enough in your application logic, no?

**Comment by atao** [comment]
I agree, it doesn't make sense to integrate this into the bookstore. I close this issue.
